### PR TITLE
The change you always wanted, removing round start cryo tubes

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request
Removes the cryo tubes, piping, and coolers from all medbays, and replaces them with empty machine frames. also the shift start cryoxadone to supply said things

# Why is this good for the game?

Medbay throwing people into tubes is bad. Do your job.
gives engineers something to do (build the machines)
gives medbay something to do (their jobs)
gives chemistry more to do (make cryoxadone)
puts a smile on my face when I see people go "waaa" because they have to put one iota of effort in
plus balance council said this was better than nerfing cryox so :)
TG did it years ago, and they are fine so
# Testing

😏 

# Wiki Documentation
i dont know if wiki had a page for that

# Changelog

:cl:  
tweak: Roundstart cryotubes are gone, and now can be built when research is completed.
/:cl:

### CODE REVIEW

**Comment by Moltijoe** [comment]
Gotta ask, how are medical supposed to deal with clone damage then?
Cryoxadone basically requires a cryotube environment so its not super reliable
Cloneoxadone is notably harder to make.
This nerfs cloning more than anything, and people already complain that medical doesnt clone enough

**Review by Aquizit — APPROVED** [review]
Mapping stuff is fine,

**Comment by Kell-E** [comment]
Cloning is a vestigal limb in the medi-sphere, clone damage may as well not exist outside of plasmemes and gibbed victims. Even then, there are a few readily avaliable solutions (On roundstart) that can fix both.

The cryo tubes are useful but they're by no means required by medical.

**Comment by azzzertyy** [comment]
keep them on gax atleast, keeps it more in line with lowpop since passive healing is more needed

**Comment by Hislyna** [comment]
Im so gonna clone everyone that dies and tell them its a skill issue when they have clone damage

**Comment by MajManatee** [comment]
> Gotta ask, how are medical supposed to deal with clone damage then? Cryoxadone basically requires a cryotube environment so its not super reliable Cloneoxadone is notably harder to make. This nerfs cloning more than anything, and people already complain that medical doesnt clone enough

Cryoxadone can still be used. Put people in a cold shower and give them it. Its easily made by chem

**Comment by MajManatee** [comment]
none of the people that are downvoting play medical...

**Comment by Hislyna** [comment]
> none of the people that are downvoting play medical...

My highest played is brig phys lol (other than unknown)

**Comment by Tenn0x** [comment]
how about some cryoxadone spawning in cloning for biomass?

**Comment by RG4ORDR** [comment]
> none of the people that are downvoting play medical...

I'm against this as someone that does play medical.

This would make sense if we consistently didn't have rounds where cryotubes weren't needed. Most of our rounds last an hour at max, some go into the near 2 hour mark occasionally. Cryotubes aren't the main way to heal either but they are extremely useful to have round start. You can't even build them til we get cryostasis tech which in my experience we have to ask to get researched more often than not.

"Oh many they have a lot of various damage that can be healed via the tube."
"I'm working on 3 other patients."
"Mass casualty events."
"I'm to do surgery but they want to go into the sleeper or the tube."
"We can't make stasis beds right now, or engineering has made us, or we haven't made stasis beds so a way to heal patients when we have limited space is to plop them into the cryo tube."

It's a loop and recycle you go through.
Sleeper, Surgery, Tube, Cloning.
Cloning, Tube.

Just because TG did it, doesn't mean it's a good idea.

**Comment by Moltijoe** [comment]
i've made #20585 
if that gets approved and merged, then honestly i'm alright with this

**Comment by Misanthropy-AU** [comment]
You could remove stasis beds from the game entirely and it wouldn't affect medical in any meaningful way. you could remove sleepers from the game entirely, and, likewise to stasis, not change anything. but cryo tubes as the game currently stands is pretty much a hard requirement for MD's to treat cellular and will need some form of round start available replacement if removed.

as molti said, cryoxadone requires extremely cold temperatures to function, and the shower simply doesn't reach the same levels the tubes do and is not at all reliable for use like that, requiring to be stacked with other chemicals such as iced coffee/tea/rhigoxane

furthermore on a personal level i'm against removing any more functionality/Capability from MD's and throwing it onto the CMO or chemists. a competent chemist already replaces the entire medical department, and the current MD experience blows.

i'm all for merging Molti's PR to enable sleepers to deal with cloning's cellular damage in place of cryo, because, seriously, i absolutely despise being shoved into cryo (especially with the cryox nerf and changes to lizards, as the tubes can pretty easily bring you back down to half or less of your health with how long it takes to recover from the cold temps currently without being fed either leporazine, coffee or being shoved under the shower immediately whilst unconscious for fucking ever), but alterations will need to be made for medical to start with one sleeper upgraded with a nano manipulator to enable it's cellular healing functionality.
adding even more hard dependencies on competent non-antag chemistry(a two slot only job btw that attracts a lot of newer playes) science, engineering and atmos (because atmos techs are required for functional cryo tubes too as they require a piped o2 or healium supply, which is not an engineers job to setup) will just have medbay largely leaving cellular damage untreated and also means it will very rarely if ever get constructed.

if we merge this without merging molti's PR, we'd likewise have to follow other servers and add a separate pharmacy area to med for MD's to make the chems they need to treat people themselves, and deal with the issues that will create.

TL;DR - i despise cryo too, but i think this is stupid & thoughtless. not even sure why cryox is being nerfed (again) in the first place. is cryo with it's hefty downsides really that big of an issue that it also needs to be removed from existence?
Molti's sleeper PR to heal cellular is also just going to be a buff to medical across the board, effectively removing any need for cryo anyway as you won't have the downsides of being temporarily slowed down by cold, knocked unconscious temporarily and constantly falling over until the cryox is out of your system.

Also, in regards to the votes just because manatee brought it up earlier.
People who Currently agree as of time of posting are: inactive players, or players who have never put enough hours into the medical department to be able to have a solid understanding of what's good, what isn't, and how things work, or mains of other external departments that aren't medical.
People who Disagree with this PR: Several active players & mentors, current medical players and some inactive medical players.

Speaks for itself, really.

**Comment by Aquizit** [comment]
Having thought about it over the last day (reminder my review approval is only about the map being functionally correct) I feel like the issue here is that people get thrown in the tube because medbay is busy or whatever, and then spend forever asleep because no one shakes them awake. Or incompetent staff don't realize that it's something that cryo doesn't fix or whatever. That is on medbay staff not the tube.

Honestly just remove sleep from cryotube tbh.

**Comment by MajManatee** [comment]
too many words sheesh

**Comment by Moltijoe** [comment]
Balance Council:
>This still doesn't fix the problem, now it just means people will sit dead or left with large amounts of cloning damage
TG removing cryo isn't a valid excuse, they effectively have permadeath with how much they've gutted medical
